code/game/machinery/cloning.dm

```diff
@@ -467,6 +467,7 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		to_chat(occupant, "<span class='userdanger'><b>You do not remember anything about the events leading up to your death.</b></span>")
```

Comment:
```suggestion
		to_chat(occupant, "<span class='userdanger'>You do not remember anything about the events leading up to your death.</span>")
```
userdanger is bold.
See stylesheet.dm
`.userdanger				{color: #ff0000;	font-weight: bold; font-size: 3;}`